<!DOCTYPE html>
<html lang="en" dir="auto">

<head><script src="/Hi1993Ryan/Hi1993Ryan.github.io/livereload.js?mindelay=10&amp;v=2&amp;port=1313&amp;path=Hi1993Ryan/Hi1993Ryan.github.io/livereload" data-no-instant defer></script><meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="noindex, nofollow">
<title>Papers | Dr. Jiangrui Liu | 刘江锐 博士</title>
<meta name="keywords" content="">
<meta name="description" content="Currently I&rsquo;m a Senior Bioinformatics Scientist at Illumina , based at their R&amp;D site near Cambridge in the UK. Here I work on solutions to key informatics hurdles in the sequencing space, such as efficient and automated bioinformatics data analysis. I enjoy solving hard problems by writing quality software, preferably leveraging modern programming languages, agile development practices and effective cloud services.
Employment 2020- Illumina, Inc. Senior Bioinformatics Scientist (software engineer and solutions architect).">
<meta name="author" content="Name">
<link rel="canonical" href="http://localhost:1313/Hi1993Ryan/Hi1993Ryan.github.io/pubs/">
<link crossorigin="anonymous" href="/Hi1993Ryan/Hi1993Ryan.github.io/assets/css/stylesheet.f1e4501a2ac2bf9fff5dc0c77f152affb825b371cb176acfcf9201015d59b4d4.css" integrity="sha256-8eRQGirCv5//XcDHfxUq/7gls3HLF2rPz5IBAV1ZtNQ=" rel="preload stylesheet" as="style">
<link rel="icon" href="http://localhost:1313/Hi1993Ryan/Hi1993Ryan.github.io/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="http://localhost:1313/Hi1993Ryan/Hi1993Ryan.github.io/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="http://localhost:1313/Hi1993Ryan/Hi1993Ryan.github.io/favicon-32x32.png">
<link rel="apple-touch-icon" href="http://localhost:1313/Hi1993Ryan/Hi1993Ryan.github.io/apple-touch-icon.png">

<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<link rel="alternate" hreflang="en" href="http://localhost:1313/Hi1993Ryan/Hi1993Ryan.github.io/pubs/">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
</noscript><meta property="og:title" content="Papers" />
<meta property="og:description" content="Currently I&rsquo;m a Senior Bioinformatics Scientist at Illumina , based at their R&amp;D site near Cambridge in the UK. Here I work on solutions to key informatics hurdles in the sequencing space, such as efficient and automated bioinformatics data analysis. I enjoy solving hard problems by writing quality software, preferably leveraging modern programming languages, agile development practices and effective cloud services.
Employment 2020- Illumina, Inc. Senior Bioinformatics Scientist (software engineer and solutions architect)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="http://localhost:1313/Hi1993Ryan/Hi1993Ryan.github.io/pubs/" /><meta property="article:section" content="" />



<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Papers"/>
<meta name="twitter:description" content="Currently I&rsquo;m a Senior Bioinformatics Scientist at Illumina , based at their R&amp;D site near Cambridge in the UK. Here I work on solutions to key informatics hurdles in the sequencing space, such as efficient and automated bioinformatics data analysis. I enjoy solving hard problems by writing quality software, preferably leveraging modern programming languages, agile development practices and effective cloud services.
Employment 2020- Illumina, Inc. Senior Bioinformatics Scientist (software engineer and solutions architect)."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [
    {
      "@type": "ListItem",
      "position":  1 ,
      "name": "Papers",
      "item": "http://localhost:1313/Hi1993Ryan/Hi1993Ryan.github.io/pubs/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "Papers",
  "name": "Papers",
  "description": "Currently I\u0026rsquo;m a Senior Bioinformatics Scientist at Illumina , based at their R\u0026amp;D site near Cambridge in the UK. Here I work on solutions to key informatics hurdles in the sequencing space, such as efficient and automated bioinformatics data analysis. I enjoy solving hard problems by writing quality software, preferably leveraging modern programming languages, agile development practices and effective cloud services.\nEmployment 2020- Illumina, Inc. Senior Bioinformatics Scientist (software engineer and solutions architect).",
  "keywords": [
    
  ],
  "articleBody": "Currently I’m a Senior Bioinformatics Scientist at Illumina , based at their R\u0026D site near Cambridge in the UK. Here I work on solutions to key informatics hurdles in the sequencing space, such as efficient and automated bioinformatics data analysis. I enjoy solving hard problems by writing quality software, preferably leveraging modern programming languages, agile development practices and effective cloud services.\nEmployment 2020- Illumina, Inc. Senior Bioinformatics Scientist (software engineer and solutions architect).\nMy work includes integrating Illumina’s best-in-class DRAGEN bioinformatics analysis pipelines with our flagship cloud platforms (BaseSpace Sequence Hub and Illumina Connected Analytics). I also develop public tools such as the BaseSpace Sequence Hub Commandline Interface, alongside many other internal software engineering projects ranging from bioinformatics scripts to web applications.\n2015-20 Illumina, Inc. Bioinformatics Scientist (data scientist and statistical programmer).\nI joined Illumina’s R\u0026D organisation in 2015 where I worked on a wide range of bioinformatics and data science projects. Some public-facing work focused on variant comparison and analysis tooling, and I also advanced the popular Platinum Genomes gold-standard variant truthset by introducing novel validation methods.\nTechnical skills R Stata LaTeX Markdown Zotero Git and GitHub HTML and CSS Education 2012-15 University of Edinburgh Bioinformatics PhD (supervisors: Colin Semple and Stuart Aitken). My thesis was titled Unravelling higher order chromatin organisation through statistical analysis and is available through the Edinburgh Research Archive .\n2011-12 Imperial College London MSc Bioinformatics and Theoretical Systems Biology (Distinction)\n2008-11 University of York BSc Biology (First class honours)\nCertifications 2021-24 AWS Certified Solutions Architect (Associate)\nPublications Journal articles 2019 Krusche P, Trigg L, Boutros PC, Mason CE, De La Vega FM, Moore BL et al. Best practices for benchmarking germline small-variant calls in human genomes. Nature Biotechnology, 37:555–56.\n2017 Peluso S, Douglas A, Hill A, De Angelis C, Moore BL et al. Fibroblast growth factors (FGFs) prime the limb specific Shh enhancer for chromatin changes that balance histone acetylation mediated by E26 transformation-specific (ETS) factors. eLife, 6:e28590.\nEberle MA, Fritzilas E, Krusche P, Källberg M, Moore BL et al. A reference data set of 5.4 million phased human variants validated by genetic inheritance from sequencing a three-generation 17-member pedigree. Genome Research, 27:157-164.\n2016 Yu N, Nützmann H-W, MacDonald JT, Moore B et al. Delineation of metabolic gene clusters in plant genomes by chromatin signatures. Nucleic Acids Research, 44(5):2255-2265.\nWelch L, Brooksbank C, Schwartz R, Morgan SL, Gaeta B, Kilpatrick AM, Meitchen D, Moore BL et al. Applying, Evaluating and Refining Bioinformatics Core Competencies. PLoS Compututational Biology, 12(5): e1004943.\n2015 Moore BL et al. Integrative modeling reveals the principles of multi-scale chromatin boundary formation in human nuclear organization. Genome Biology, 16:110.\nFraser J, Ferrai C, Chiariello AM, Schueler M, Rito T, Laudanno G, Barbieri M, Moore BL et al. Hierarchical folding and reorganization of chromosomes are linked to transcriptional changes in cellular differentiation. Molecular systems biology, 11:852.\n2013 Moore BL et al. High–quality protein backbone reconstruction from alpha carbons using Gaussian mixture models. Journal of computational chemisty, 34(22):1881-9.\n2011 Gardner PP, Daub J, Tate J, Moore BL et al. Rfam: Wikipedia, clans and the “decimal” release. Nucleic acids research, 39(S1):D141-5.\nBook chapters 2017 Moore BL, Semple CA. Crossing Borders: Modeling Approaches to Understand Chromatin Domains and Their Boundaries. Epigenetics and Systems Biology, 3:45-67.\nNon-academic 2014 Pieces in The Huffington Post (US): Here Are the Most Overrated and Underrated Movies of All Time; Celebrity Twitter Followers, by Gender (huffingtonpost.com/benjamin-moore )\n2006– Articles for English Wikipedia: European Nucleotide Archive, RNA thermometer, Toxin-antitoxin system and more (en.wikipedia.org/wiki/User:Ben_Moore ).\nProjects github My personal github account hosts my hobby development projects as well as listing contributions to open source tools. Examples include: shiny apps blackspot and shinybrot , as well as an R package hosted by CRAN (mandelbrot ).\nblog I write a data science blog at blm.io where I apply R and Python to open datasets and write-up the results. My posts have been featured by sites including FiveThirtyEight, BuzzFeed, AVclub, io9 and more, and also led to being an invited blogger at the Huffington Post. My original wordpress blog (benjaminlmoore.wordpress.com ) has received over 150,000 unique visitors from 180 countries.\nEdinbR I co-founded a usergroup for the R programming language in Edinburgh called EdinbR. I helped to organise our monthly meetings and built our website: edinbr.org . Our meetings attract 30-50 statisticians, data scientists and developers for talks and discussion about the R language and its applications. We’re sponsored by Revolution Analytics and have an organisational github account at github.com/EdinbR .\ndatarea I entered Imperial College’s Summer Data Challenge competition, where entrants analysed a given dataset and then proposed a startup idea based on their results. I developed a modelling technique to select housing areas for investment and was awarded third place: £2,000 and startup support from Imperial Create Lab. My entry can be seen at blm.io/datarea and the code is shared on my github account.\nAwards MRC Capacity building scholarship (PhD fees + enhanced stipend)\nMRC Full postgraduate scholarship (MSc fees + stipend)\nISCB computational biology Wikipedia competition 2013 (doi:10.1371/journal.pcbi.1003242 ; 2nd place)\nImperial Create Lab’s “Summer data challenge” (summerdatachallenge.com ; 3rd place)\nIndividual engagement grant from the Wikimedia Foundation to attend ISMB 2014\nTraining accreditation for Wikimedia UK, the national chapter of the Wikimedia Foundation\nFront cover design selected for J. Comp. Chem. (34:22) (doi:10.1002/jcc.23388 )\nPlacements 2010-11 Wellcome Trust Sanger Institute (Cambridge, UK) Worked for two summers on the Rfam database (rfam.xfam.org ) in the Bateman group\nLinks ben@blm.io blmoore\nbenjaminlmoore\nLinkedIn StackOverflow Wikipedia Userpage\nGoogle scholar ORCiD figshare References Available on request.\n",
  "wordCount" : "919",
  "inLanguage": "en",
  "datePublished": "0001-01-01T00:00:00Z",
  "dateModified": "0001-01-01T00:00:00Z",
  "author":{
    "@type": "Person",
    "name": "Name"
  },
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "http://localhost:1313/Hi1993Ryan/Hi1993Ryan.github.io/pubs/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "Dr. Jiangrui Liu | 刘江锐 博士",
    "logo": {
      "@type": "ImageObject",
      "url": "http://localhost:1313/Hi1993Ryan/Hi1993Ryan.github.io/favicon.ico"
    }
  }
}
</script>



<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/katex.min.css" integrity="sha384-RZU/ijkSsFbcmivfdRBQDtwuwVqK7GMOw6IMvKyeWL2K5UAlyp6WonmB8m7Jd0Hn" crossorigin="anonymous">
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/katex.min.js" integrity="sha384-pK1WpvzWVBQiP0/GjnvRxV4mOb0oxFuyRxJlk6vVw146n3egcN5C925NCP7a7BY8" crossorigin="anonymous"></script>
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/contrib/auto-render.min.js" integrity="sha384-vZTG03m+2yp6N6BNi5iM4rW4oIwk5DfcNdFfxkk9ZWpDriOkXX8voJBFrAO7MpVl" crossorigin="anonymous"></script>
<script>
    document.addEventListener("DOMContentLoaded", function() {
        renderMathInElement(document.body, {
          
          
          delimiters: [
            {left: '$$', right: '$$', display: true},
            {left: '$', right: '$', display: false},
            {left: "\\begin{equation}", right: "\\end{equation}", display: true},
            {left: "\\begin{equation*}", right: "\\end{equation*}", display: true},
            {left: "\\begin{align}", right: "\\end{align}", display: true},
            {left: "\\begin{align*}", right: "\\end{align*}", display: true},
            {left: "\\begin{alignat}", right: "\\end{alignat}", display: true},
            {left: "\\begin{gather}", right: "\\end{gather}", display: true},
            {left: "\\begin{CD}", right: "\\end{CD}", display: true},
          ],
          
          throwOnError : false
        });
    });
</script>
 


</head>

<body class="" id="top">

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="http://localhost:1313/Hi1993Ryan/Hi1993Ryan.github.io/" accesskey="h" title="Dr. Jiangrui Liu">
             
                <img src="http://localhost:1313/Hi1993Ryan/Hi1993Ryan.github.io/favicon.ico" alt="" aria-label="logo"
                    height="18"
                    width="18">Dr. Jiangrui Liu</a>
            <div class="logo-switches">
            </div>
        </div>
        <ul id="menu">
            <li>
                <a href="http://localhost:1313/Hi1993Ryan/Hi1993Ryan.github.io/pubs/" title="Papers">
                    <span class="active">Papers</span>
                </a>
            </li>
            <li>
                <a href="http://localhost:1313/Hi1993Ryan/Hi1993Ryan.github.io/courses/" title="Courses">
                    <span>Courses</span>
                </a>
            </li>
            <li>
                <a href="http://localhost:1313/Hi1993Ryan/Hi1993Ryan.github.io/projects/" title="Projects">
                    <span>Projects</span>
                </a>
            </li>
        </ul>
    </nav>
</header>

    <main class="main">

<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      Papers
    </h1>
    <div class="post-meta">Name

</div>
  </header> 
  <div class="post-content"><h2 id="currently">Currently</h2>
<p>I&rsquo;m a Senior Bioinformatics Scientist at <a href="http://www.illumina.com/" target="_blank">Illumina</a>
, based at their R&amp;D site near Cambridge in the UK. Here I work on solutions to key informatics hurdles in the sequencing space, such as efficient and automated bioinformatics data analysis. I enjoy solving hard problems by writing quality software, preferably leveraging modern programming languages, agile development practices and effective cloud services.</p>
<h2 id="employment">Employment</h2>
<p><code>2020-</code>
<strong>Illumina, Inc.</strong> Senior Bioinformatics Scientist (software engineer and solutions architect).</p>
<blockquote>
<p>My work includes integrating Illumina&rsquo;s best-in-class <em>DRAGEN</em> bioinformatics analysis pipelines with our flagship cloud platforms (<em>BaseSpace Sequence Hub</em> and <em>Illumina Connected Analytics</em>). I also develop public tools such as the BaseSpace Sequence Hub Commandline Interface, alongside many other internal software engineering projects ranging from bioinformatics scripts to web applications.</p>
</blockquote>
<p><code>2015-20</code>
<strong>Illumina, Inc.</strong> Bioinformatics Scientist (data scientist and statistical programmer).</p>
<blockquote>
<p>I joined Illumina&rsquo;s R&amp;D organisation in 2015 where I worked on a wide range of bioinformatics and data science projects. Some public-facing work focused on variant comparison and analysis tooling, and I also advanced the popular <em>Platinum Genomes</em> gold-standard variant truthset by introducing novel validation methods.</p>
</blockquote>
<h2 id="technical-skills">Technical skills</h2>
<ul>
<li>R</li>
<li>Stata</li>
<li><em>LaTeX</em></li>
<li>Markdown</li>
<li>Zotero</li>
<li>Git and GitHub</li>
<li>HTML and CSS</li>
</ul>
<h2 id="education">Education</h2>
<p><code>2012-15</code>
<strong>University of Edinburgh</strong> Bioinformatics PhD (supervisors: Colin Semple and Stuart Aitken). My thesis was titled <em>Unravelling higher order chromatin organisation through statistical analysis</em> and is available through the <a href="https://www.era.lib.ed.ac.uk/handle/1842/22906" target="_blank">Edinburgh Research Archive</a>
.</p>
<p><code>2011-12</code>
<strong>Imperial College London</strong> MSc Bioinformatics and Theoretical Systems Biology (<em>Distinction</em>)</p>
<p><code>2008-11</code>
<strong>University of York</strong> BSc Biology (<em>First class honours</em>)</p>
<h3 id="certifications">Certifications</h3>
<p><code>2021-24</code> <strong>AWS Certified Solutions Architect</strong> (<em>Associate</em>)</p>
<h2 id="--areas-of-expertise--"><!--Areas of expertise--></h2>
<ul>
<li>
<!--Bioinformatics-->
</li>
<li>
<!--Software development-->
</li>
<li>
<!--Cloud engineering and automation-->
</li>
<li>
<!--Single-page applications-->
</li>
<li>
<!--Population genomics-->
</li>
<li>
<!--Machine learning-->
</li>
<li>
<!--Data visualisation-->
</li>
<li>
<!--Variant calling-->
</li>
<li>
<!--Illumina sequencing data-->
</li>
</ul>
<h2 id="publications">Publications</h2>
<h3 id="journal-articles">Journal articles</h3>
<p><code>2019</code>
Krusche P, Trigg L, Boutros PC, Mason CE, De La Vega FM, <strong>Moore BL</strong> <em>et al.</em> Best practices for benchmarking germline small-variant calls in human genomes. <em>Nature Biotechnology</em>, 37:555–56.</p>
<p><code>2017</code>
Peluso S, Douglas A, Hill A, De Angelis C, <strong>Moore BL</strong> <em>et al.</em> Fibroblast growth factors (FGFs) prime the limb specific Shh enhancer for chromatin changes that balance histone acetylation mediated by E26 transformation-specific (ETS) factors. <em>eLife</em>, 6:e28590.</p>
<p>Eberle MA, Fritzilas E, Krusche P, Källberg M, <strong>Moore BL</strong> <em>et al.</em> A reference data set of 5.4 million phased human variants validated by genetic inheritance from sequencing a three-generation 17-member pedigree. <em>Genome Research</em>, 27:157-164.</p>
<p><code>2016</code>
Yu N, Nützmann H-W, MacDonald JT, <strong>Moore B</strong> <em>et al.</em> Delineation of metabolic gene clusters in plant genomes by chromatin signatures. <em>Nucleic Acids Research</em>, 44(5):2255-2265.</p>
<p>Welch L, Brooksbank C, Schwartz R, Morgan SL, Gaeta B, Kilpatrick AM, Meitchen D, <strong>Moore BL</strong> <em>et al.</em> Applying, Evaluating and Refining Bioinformatics Core Competencies. <em>PLoS Compututational Biology</em>, 12(5): e1004943.</p>
<p><code>2015</code>
<strong>Moore BL</strong> <em>et al.</em> Integrative modeling reveals the principles of multi-scale chromatin boundary formation in human nuclear organization. <em>Genome Biology</em>, 16:110.</p>
<p>Fraser J, Ferrai C, Chiariello AM, Schueler M, Rito T, Laudanno G, Barbieri M, <strong>Moore BL</strong> <em>et al.</em> Hierarchical folding and reorganization of chromosomes are linked to transcriptional changes in cellular differentiation. <em>Molecular systems biology</em>, 11:852.</p>
<p><code>2013</code>
<strong>Moore BL</strong> <em>et al.</em> High–quality protein backbone reconstruction from alpha carbons using Gaussian mixture models. <em>Journal of computational chemisty</em>, 34(22):1881-9.</p>
<p><code>2011</code>
Gardner PP, Daub J, Tate J, <strong>Moore BL</strong> <em>et al.</em> Rfam: Wikipedia, clans and the “decimal” release. <em>Nucleic acids research</em>, 39(S1):D141-5.</p>
<h3 id="book-chapters">Book chapters</h3>
<p><code>2017</code>
<strong>Moore BL</strong>, Semple CA. Crossing Borders: Modeling Approaches to Understand Chromatin Domains and Their Boundaries. <em>Epigenetics and Systems Biology</em>, 3:45-67.</p>
<h2 id="non-academic">Non-academic</h2>
<p><code>2014</code> Pieces in <em>The Huffington Post</em> (US): Here Are the Most Overrated and Underrated Movies of All Time; Celebrity Twitter Followers, by Gender (<a href="http://www.huffingtonpost.com/benjamin-moore/" target="_blank">huffingtonpost.com/benjamin-moore</a>
)</p>
<p><code>2006–</code> Articles for <em>English Wikipedia</em>: European Nucleotide Archive, RNA thermometer, Toxin-antitoxin system and more (<a href="https://en.wikipedia.org/wiki/User:Ben_Moore" target="_blank">en.wikipedia.org/wiki/User:Ben_Moore</a>
).</p>
<h2 id="projects">Projects</h2>
<h3 id="github">github</h3>
<p>My <a href="https://github.com/blmoore" target="_blank">personal github account</a>
 hosts my hobby development projects as well as listing contributions to open source tools. Examples include: shiny apps <a href="https://github.com/blmoore/blackspot" target="_blank">blackspot</a>
 and <a href="https://github.com/blmoore/shinybrot" target="_blank">shinybrot</a>
, as well as an R package hosted by CRAN (<a href="https://github.com/blmoore/mandelbrot" target="_blank">mandelbrot</a>
).</p>
<h3 id="blog">blog</h3>
<p>I write a data science blog at <a href="http://blm.io/blog" target="_blank">blm.io</a>
 where I apply R and Python to open datasets and write-up the results. My posts have been featured by sites including FiveThirtyEight, BuzzFeed, AVclub, io9 and more, and also led to being an invited blogger at the Huffington Post. My original wordpress blog (<a href="http://benjaminlmoore.wordpress.com" target="_blank">benjaminlmoore.wordpress.com</a>
) has received over 150,000 unique visitors from 180 countries.</p>
<h3 id="edinbr">EdinbR</h3>
<p>I co-founded a usergroup for the R programming language in Edinburgh called EdinbR. I helped to organise our monthly meetings and built our website: <a href="http://edinbr.org" target="_blank">edinbr.org</a>
. Our meetings attract 30-50 statisticians, data scientists and developers for talks and discussion about the R language and its applications. We&rsquo;re sponsored by Revolution Analytics and have an organisational github account at <a href="https://github.com/EdinbR" target="_blank">github.com/EdinbR</a>
.</p>
<h3 id="datarea">datarea</h3>
<p>I entered Imperial College&rsquo;s <a href="https://www.imperial.ac.uk/data-science/education/summer-data-challenge/" target="_blank">Summer Data Challenge</a>
 competition, where entrants analysed a given dataset and then proposed a startup idea based on their results. I developed a modelling technique to select housing areas for investment and was awarded third place: £2,000 and startup support from Imperial Create Lab. My entry can be seen at <a href="http://blm.io/datarea" target="_blank">blm.io/datarea</a>
 and the code is shared on my github account.</p>
<h2 id="awards">Awards</h2>
<p>MRC Capacity building scholarship (PhD fees + enhanced stipend)</p>
<p>MRC Full postgraduate scholarship (MSc fees + stipend)</p>
<p>ISCB computational biology Wikipedia competition 2013 (<a href="http://dx.doi.org/10.1371/journal.pcbi.1003242" target="_blank">doi:10.1371/journal.pcbi.1003242</a>
; 2nd place)</p>
<p>Imperial Create Lab&rsquo;s &ldquo;Summer data challenge&rdquo; (<a href="http://summerdatachallenge.com" target="_blank">summerdatachallenge.com</a>
; 3rd place)</p>
<p>Individual engagement grant from the Wikimedia Foundation to attend ISMB 2014</p>
<p>Training accreditation for Wikimedia UK, the national chapter of the Wikimedia Foundation</p>
<p>Front cover design selected for J. Comp. Chem. (34:22) (<a href="http://dx.doi.org/10.1002/jcc.23388" target="_blank">doi:10.1002/jcc.23388</a>
)</p>
<h2 id="placements">Placements</h2>
<p><code>2010-11</code>
<strong>Wellcome Trust Sanger Institute</strong> (<em>Cambridge, UK</em>)
Worked for two summers on the Rfam database (<a href="http://rfam.xfam.org" target="_blank">rfam.xfam.org</a>
) in the Bateman group</p>
<h2 id="links">Links</h2>
<!-- fa are fontawesome, ai are academicons -->
<ul>
<li><i class="fa fa-envelope"></i> <a href="mailto:ben@blm.io"><a href="mailto:ben@blm.io">ben@blm.io</a>
</a><br /></li>
<li><i class="fa fa-github"></i> <a href="http://github.com/blmoore">blmoore</a><br /></li>
<li><i class="fa fa-twitter"></i> <a href="http://twitter.com/benjaminlmoore">benjaminlmoore</a><br /></li>
<li><i class="fa fa-linkedin"></i> <a href="https://www.linkedin.com/in/blmoore/">LinkedIn</a></li>
<li><i class="fa fa-stack-overflow"></i> <a href="http://stackoverflow.com/users/1274516/blmoore">StackOverflow</a></li>
<li><i class="fa fa-wikipedia"></i> <a href="https://en.wikipedia.org/wiki/User:Ben_Moore">Wikipedia Userpage</a><br /></li>
<li><i class="ai ai-google-scholar"></i> <a href="http://scholar.google.com/citations?user=YMxsGpsAAAAJ">Google scholar</a></li>
<li><i class="ai ai-orcid"></i> <a href="https://orcid.org/0000-0002-4074-1933">ORCiD</a></li>
<li><i class="ai ai-figshare"></i> <a href="https://figshare.com/authors/Benjamin_Moore/99461">figshare</a></li>
</ul>
<h2 id="references">References</h2>
<p>Available on request.</p>
<!-- ### Footer

Last updated: May 2013 -->

  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    &copy; 2024 Dr. Jiangrui Liu | 刘江锐 博士
    <span>
   
    </span>
</footer>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    document.querySelectorAll('pre > code').forEach((codeblock) => {
        const container = codeblock.parentNode.parentNode;

        const copybutton = document.createElement('button');
        copybutton.classList.add('copy-code');
        copybutton.innerHTML = 'copy';

        function copyingDone() {
            copybutton.innerHTML = 'copied!';
            setTimeout(() => {
                copybutton.innerHTML = 'copy';
            }, 2000);
        }

        copybutton.addEventListener('click', (cb) => {
            if ('clipboard' in navigator) {
                navigator.clipboard.writeText(codeblock.textContent);
                copyingDone();
                return;
            }

            const range = document.createRange();
            range.selectNodeContents(codeblock);
            const selection = window.getSelection();
            selection.removeAllRanges();
            selection.addRange(range);
            try {
                document.execCommand('copy');
                copyingDone();
            } catch (e) { };
            selection.removeRange(range);
        });

        if (container.classList.contains("highlight")) {
            container.appendChild(copybutton);
        } else if (container.parentNode.firstChild == container) {
            
        } else if (codeblock.parentNode.parentNode.parentNode.parentNode.parentNode.nodeName == "TABLE") {
            
            codeblock.parentNode.parentNode.parentNode.parentNode.parentNode.appendChild(copybutton);
        } else {
            
            codeblock.parentNode.appendChild(copybutton);
        }
    });
</script>
</body>
</html>
